src/Pos/Explorer/Txp/Toil/Class.hs

```diff
@@ -38,7 +39,7 @@ class Monad m => MonadTxExtraRead m where
 instance MonadTxExtraRead m => MonadTxExtraRead (ReaderT s m)
 instance MonadTxExtraRead m => MonadTxExtraRead (StateT s m)
 instance MonadTxExtraRead m => MonadTxExtraRead (ExceptT s m)
-instance MonadTxExtraRead m => MonadTxExtraRead (UtxoReaderT m)
+instance MonadTxExtraRead m => MonadTxExtraRead (Ether.ReaderT s m)
```

Comment:
Write a single "catch-all" lifting instance:

```
instance {-# OVERLAPPABLE #-}
    (MonadTxExtraRead m, MonadTrans t, Monad (t m)) =>
        MonadTxExtraRead (t m)
```

This way you don't need to write an instance per transformer.

I applied this refactoring to many classes in #277 but there's still some remaining.